docs/reference/setup/install.asciidoc

```diff
@@ -87,6 +87,10 @@ bundled JVM is treated as an integral part of {es}, which means that Elastic
 takes responsibility for keeping it up to date. Security issues and bugs within
 the bundled JVM are treated as if they were within {es} itself.
 
+You should not use long-running third-party agents that attach to the JVM
```

Comment:
Remove "long-running"? Any java agent, regardless of how long lived, should not be attached to Elasticsearch.